pkg/util/mount/mount_linux.go

```diff
@@ -54,7 +54,9 @@ const (
 	// place for subpath mounts
 	containerSubPathDirectoryName = "volume-subpaths"
 	// syscall.Openat flags used to traverse directories not following symlinks
-	nofollowFlags = syscall.O_RDONLY | syscall.O_NOFOLLOW
+	nofollowFlags = unix.O_RDONLY | unix.O_NOFOLLOW
```

Comment:
Do we still use this anywhere?

Reply:
Yes, in few places with outer `Open` calls when we are opening base directory.